requirements.txt

```diff
@@ -2,6 +2,7 @@
 Flask==1.1.1
 Flask-API==1.1
 Flask-SQLAlchemy==2.4.1
+flask-restplus==0.13.0
```

Comment:
this has already been defined. we can remove this

Reply:
True, was due to the merge conflict.

Reply:
yep. Thanks